Quarkus Tools (1.11.1): Internal error: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/intellij/openapi/roots/impl/ProjectFileIndexImpl.getModuleForFile must not be null

Exception in plugin Quarkus Tools (1.11.1.238)

My environment
```
IntelliJ IDEA 2022.1.2 (Ultimate Edition)
Build #IU-221.5787.30, built on May 31, 2022
Runtime version: 11.0.15+10-b2043.56 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-113-generic

Non-Bundled Plugins:
    org.rust.lang (0.4.171.4656-221)
    org.jetbrains.plugins.go (221.5787.20)
    dev.meanmail.plugin.nginx-intellij-plugin (2021.3-2022.1)
    com.redhat.devtools.intellij.telemetry (0.0.2.18)
    com.intellij.ideolog (203.0.30.0)
    org.jetbrains.plugins.hocon (2022.1.0)
    com.jetbrains.jax.ws (221.5787.20)
    com.intellij.guice (221.5787.20)
    VisualVMLauncher (1.21.211.6085.0)
    GrepConsole (12.12.211.6693.0)
    org.sonarlint.idea (6.7.0.45926)
    mobi.hsz.idea.gitignore (4.4.0)
    com.intellij.kubernetes (221.5787.15)
    CMD Support (1.0.5)
    Batch Scripts Support (1.0.13)
    com.redhat.devtools.intellij.quarkus (1.11.1.238)
    MavenRunHelper (4.20.213.3714.1)
    com.intellij.plugins.html.instantEditing (221.5787.15)
    com.dmarcotte.handlebars (221.5787.30)
    com.emberjs (2022.1.2)
    ru.adelf.idea.dotenv (2022.1)

Kotlin: 221-1.6.21-release-337-IJ5787.30
Current Desktop: KDE
```

```
Internal error: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/intellij/openapi/roots/impl/ProjectFileIndexImpl.getModuleForFile must not be null

java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/intellij/openapi/roots/impl/ProjectFileIndexImpl.getModuleForFile must not be null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:346)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:704)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2088)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.runTask(QuteLanguageClient.java:131)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.access$000(QuteLanguageClient.java:64)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient$1.run(QuteLanguageClient.java:106)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressInCurrentThread$10(CoreProgressManager.java:561)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:189)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:176)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressInCurrentThread(CoreProgressManager.java:561)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:381)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.lambda$runAsBackground$2(QuteLanguageClient.java:103)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.lambda$runAsBackground$3(QuteLanguageClient.java:117)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1728)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/intellij/openapi/roots/impl/ProjectFileIndexImpl.getModuleForFile must not be null
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.$$$reportNull$$$0(ProjectFileIndexImpl.java)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.getModuleForFile(ProjectFileIndexImpl.java)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.internal.core.ls.PsiUtilsLSImpl.getModule(PsiUtilsLSImpl.java:88)
	at com.redhat.devtools.intellij.qute.psi.QuteSupportForTemplate.getJavaProjectFromTemplateFile(QuteSupportForTemplate.java:523)
	at com.redhat.devtools.intellij.qute.psi.QuteSupportForJava.diagnostics(QuteSupportForJava.java:70)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.lambda$getJavaDiagnostics$15(QuteLanguageClient.java:178)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:933)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:68)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.lambda$getJavaDiagnostics$16(QuteLanguageClient.java:178)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.runTask(QuteLanguageClient.java:127)
	... 23 more

```

--- a/src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/core/ls/PsiUtilsLSImpl.java
+++ b/src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/core/ls/PsiUtilsLSImpl.java
@@ -84,11 +84,8 @@ public class PsiUtilsLSImpl implements IPsiUtils {
 
     @Override
     public Module getModule(VirtualFile file) {
-        for (Project project : ProjectManager.getInstance().getOpenProjects()) {
-            Module module = ProjectFileIndex.getInstance(project).getModuleForFile(file);
-            if (module != null) {
-                return module;
-            }
+        if (file != null) {
+            return ProjectFileIndex.getInstance(project).getModuleForFile(file);
         }
         return null;
     }

